stablehlo/tools/CMakeLists.txt

```diff
@@ -29,9 +29,11 @@ set(LIBS
         StablehloRegister
         StablehloTestUtils
         StablehloPasses
+        StablehloTOSATransforms
         )
 add_llvm_executable(stablehlo-opt StablehloOptMain.cpp)
 llvm_update_compile_flags(stablehlo-opt)
+add_dependencies(stablehlo-opt StablehloTOSATransformsPassIncGen)
```

Comment:
I'm not an expert in CMake, but I think this dependency might be unnecessary given that `StablehloTOSATransforms` already DEPENDS on `StablehloTOSATransformsPassIncGen`.